tests/integ/scala/test_dataframe_suite.py

```diff
@@ -1520,7 +1518,6 @@ def test_createDataFrame_with_given_schema(session):
             StructField("number", DecimalType(10, 3)),
             StructField("boolean", BooleanType()),
             StructField("binary", BinaryType()),
-            StructField("timestamp", TimestampType()),
             StructField("timestamp_ntz", TimestampType(TimestampTimeZone.NTZ)),
```

Comment:
Here setting timestamp with default timezone value currently maps to NTZ but can cause flakiness in the future so I removed it in favor of testing of other explicitly defined timezones.

Reply:
Hmm but how will we cover the default use case?

Reply:
I can try to use statement params to set default timezone to be ntz and run the test using that